### ISSUE TITLE
Unexpected cuda error

### ISSUE THREAD

**Issue by Lotayou** [AliaksandrSiarohin/pose-gan#8]
I run into this error message when trying to train a new model with Market dataset, following the exact steps on your github page:

`
2018-07-18 00:25:07.908521: E C:\Users\User\Source\Repos\tensorflow\tensorflow\stream_executor\cuda\cuda_driver.cc:1080] failed to synchronize the stop event: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018-07-18 00:25:07.908824: E C:\Users\User\Source\Repos\tensorflow\tensorflow\stream_executor\cuda\cuda_timer.cc:54] Internal: error destroying CUDA event in context 0000026F6FC0BCB0: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018-07-18 00:25:07.909146: E C:\Users\User\Source\Repos\tensorflow\tensorflow\stream_executor\cuda\cuda_timer.cc:59] Internal: error destroying CUDA event in context 0000026F6FC0BCB0: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018-07-18 00:25:07.909478: F C:\Users\User\Source\Repos\tensorflow\tensorflow\stream_executor\cuda\cuda_dnn.cc:2328] failed to set stream for cudnn handle: CUDNN_STATUS_MAPPING_ERROR
`

I managed to pinpoint the location this error occurs. It seems to be this line in `gan/train.py`:
`loss = self.discriminator_model.train_on_batch(
                            discrimiantor_batch + generator_batch, np.zeros([self.batch_size]))`

My environment is as follows:
System: Windows 10
cuda: v9.1
cudnn: v7.1
tensorflow-gpu: 1.7.0
keras: 2.2.0

I'm not sure how to fix this problem and I just wonder if you have you ever run into this problem? Also, can you tell me your configurations? Thanks!

**Comment by AliaksandrSiarohin** [comment]
I never saw, an error like this.
Can you train usual keras models like this one (https://github.com/keras-team/keras/blob/master/examples/cifar10_cnn.py)?

**Comment by Lotayou** [comment]
Well I can run cifar smoothly without any problem, I've changed several Tensorflow and cuda versions but the error still occurs, perhaps next step I'll try on Linux server and see if the error is Windows-dependent. Anyway, thanks for the tip

**Comment by xihuan1990** [comment]
I also encountered this problem, how can you solve it?
